platform/product-management/release-plan-template.md

```diff
@@ -6,6 +6,38 @@ So: **how might we craft a release plan to test our product "in the wild" at a s
 
 That's what this Release Plan Template is for! And note - there are feature toggles and beta banners at your disposal that you can use as a part of your plan.
```

Comment:
Do we still offer "beta banners" as a formal tool that folks can use? Haven't heard about anyone using that recently.